decidim-proposals/app/views/decidim/proposals/proposals/_endorsements_card_row.html.erb

```diff
@@ -0,0 +1,22 @@
+<% fully_endorsed= fully_endorsed?(@proposal, current_user) %>
+<div class="row collapse buttons__row">
+  <% if endorsements_enabled? %>
+    <div class="column small-9 collapse">
+      <div class="button-group button-group--collapse button--nomargin small">
+        <%= render partial: "endorsements_count", locals: { proposal: @proposal, fully_endorsed: fully_endorsed } %>
+        <% if current_settings.endorsements_blocked? %>
+          <%= content_tag :span, t('.endorse'), class: "card__button button #{endorsement_button_classes(false)} disabled", disabled: true, title: t('.endorse') %>
+        <% elsif not current_user %>
```

Comment:
`elsif not current_user` it's kinds complicated, maybe we can use `elsif current_user` and swap the block with the next one?

Reply:
Yeah, done.